tests/SecuritySlackTest.php

```diff
@@ -12,6 +11,10 @@ class SecuritySlackTest extends TestCase
      */
     public function testSlackNotification()
     {
+        if (! class_exists(Illuminate\Notifications\Messages\SlackMessage::class)) {
+            $this->markTestSkipped();
```

Comment:
```suggestion
            $this->markTestSkipped('this test is skipped because package `laravel/slack-notification-channel` is not installed.');
```